rosa-hypershift/rosa-hosted-wrapper.py

```diff
@@ -0,0 +1,972 @@
+    starting_time = datetime.datetime.utcnow().timestamp()
+    logging.debug("Waiting %d minutes for nodes to be Ready on cluster %s" % (wait_time, cluster_name))
+    while datetime.datetime.utcnow().timestamp() < starting_time + wait_time * 60:
+        logging.info('Getting node information for cluster %s' % cluster_name)
+        nodes_command = ["oc", "get", "nodes", "-o", "json"]
+        logging.debug(nodes_command)
+        nodes_process = subprocess.Popen(nodes_command, stdout=subprocess.PIPE, stderr=subprocess.PIPE, universal_newlines=True, env=myenv)
+        nodes_stdout, nodes_stderr = nodes_process.communicate()
+        try:
+            nodes_json = json.loads(nodes_stdout)
+        except Exception as err:
+            logging.error("Cannot load command result for cluster %s" % cluster_name)
+            logging.error(err)
+            continue
+        nodes = nodes_json['items'] if 'items' in nodes_json else []
+        status = []
+        for node in nodes:
+            conditions = node['status']['conditions'] if 'status' in node and 'conditions' in node['status'] else []
+            for condition in conditions:
+                if 'type' in condition and condition['type'] == 'Ready':
+                    status.append(condition['status'])
+        status_list = {i: status.count(i) for i in status}
+        ready_nodes = status_list['True'] if 'True' in status_list else 0
+        if ready_nodes == worker_nodes:
+            logging.info("Found %d Ready nodes on cluster %s. Expected: %d. Stopping wait." % (ready_nodes, cluster_name, worker_nodes))
+            return ready_nodes
+        else:
+            logging.info("Found %d Ready nodes on cluster %s. Expected: %d. Waiting 15 seconds for next check..." % (ready_nodes, cluster_name, worker_nodes))
+            time.sleep(15)
+    logging.error("Waiting time expired. After %d minutes there are %d ready nodes (Expected: %d) on cluster %s" % (wait_time, ready_nodes, worker_nodes, cluster_name))
+    return ready_nodes
+
+
+def _cluster_load(kubeconfig, my_path, hosted_cluster_name, mgmt_cluster_name, load_duration, jobs, es_url):
+    load_env = os.environ.copy()
+    load_env["KUBECONFIG"] = kubeconfig
+    logging.info('Cloning e2e-benchmarking repo https://github.com/cloud-bulldozer/e2e-benchmarking.git')
+    Repo.clone_from("https://github.com/cloud-bulldozer/e2e-benchmarking.git", my_path + '/e2e-benchmarking')
+    os.chdir(my_path + '/e2e-benchmarking/workloads/kube-burner')
+    load_env["JOB_ITERATIONS"] = str(jobs)
+    load_env["CHURN"] = "true"
+    load_env["CHURN_DURATION"] = load_duration
+    load_env["CHURN_PERCENT"] = "10"
+    load_env["CHURN_WAIT"] = "30s"
+    load_env["JOB_TIMEOUT"] = "6h"
+    load_env["CLEANUP_WHEN_FINISH"] = "true"
+    load_env["INDEXING"] = "true"
+    load_env["HYPERSHIFT"] = "true"
+    load_env["MGMT_CLUSTER_NAME"] = mgmt_cluster_name
+    load_env["HOSTED_CLUSTER_NS"] = "clusters-" + hosted_cluster_name
```

Comment:
Hosted Cluster namespace format is different now,  
```
ocm-staging-216r2b3oor0hlrppdmffa0sd1sfgq7j7-mrnd-fr3-0001
```
so lets do regex formats `load_env["HOSTED_CLUSTER_NS"] = ".*-" + hosted_cluster_name`